src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -94,28 +109,96 @@ following to integrate with your `my_flutter` app:
 
 3. Run `pod install`.
 
-Whenever you change the Flutter plugin dependencies in `some/path/my_flutter/pubspec.yaml`,
-you need to run `flutter pub get` from `some/path/my_flutter` to refresh the list
+{{site.alert.note}}
+When you change the Flutter plugin dependencies in `my_flutter/pubspec.yaml`,
+run `flutter pub get` in your Flutter module directory to refresh the list
 of plugins read by the `podhelper.rb` script. Then run `pod install` again from
-`some/path/MyApp`.
+in your application at`some/path/MyApp`.
+{{site.alert.end}}
+
+The `podhelper.rb` script embeds your plugins, `Flutter.framework`, and
+`App.framework` into your project.
+
+You should now be able to build the project using `⌘B`.
+
+### Embed frameworks in Xcode
+
+Alternatively, you can generate the necessary frameworks and embed them in your application
+by manually editing your existing Xcode project. You may choose to do this if members of your
+team cannot locally install Flutter SDK and CocoaPods, or if you do not wish to use CocoaPods
+as a dependency manager in your existing applications.
```

Comment:
Mention that the downside is that you have to run `flutter build ios-framework` each time you change things in Flutter